cli/args.go

```diff
@@ -142,6 +142,18 @@ func filterTerraformExtraArgs(terragruntOptions *options.TerragruntOptions, terr
 	return out
 }
 
+func filterTerraformEnvVars(terragruntOptions *options.TerragruntOptions, terragruntConfig *config.TerragruntConfig) map[string]string {
```

Comment:
Shouldn't this method only add env vars for specific commands? See `filterTerraformExtraArgs`.